lib/pages/events_page.dart

```diff
@@ -39,11 +58,27 @@ class _EventsPageState extends State<EventsPage> {
 
   Widget listSeperatorBuilder(_, __) => const Divider(height: 1);
 
+  Widget renderList(List<Event> list) {
+    final EdgeInsets safePadding = MediaQuery.of(context).padding;
+    final listEdgeInsets = EdgeInsets.fromLTRB(0, 10.0, 0, safePadding.bottom);
+
+    if (list.length > 0) {
```

Comment:
I guess this displays while list is loading for the first time or refreshing also.
To show "no events found" in that case is kind of wrong imo.
We could check the event providers `refreshing` property to not to show this.

Reply:
I don't know if refreshing property is working properly tho :) Never used it. Hope it works.

Reply:
When we refresh the list it doesn't show the `No Events` screen.